package.json

```diff
@@ -109,6 +109,7 @@
     "@types/express": "^4.17.13",
     "@types/express-session": "^1.17.4",
     "@types/i18next-fs-backend": "^1.1.1",
+    "@types/jsdom": "^21.1.7",
```

Comment:
Hi @SophiaDWokoma - Do we need to introduce this dependency in this PR? I can't see any of the introduced code making use of `jsdom`.